plugins/quay/README.md

```diff
@@ -67,3 +81,35 @@ yarn start-backend
 ```
 yarn workspace @janus-idp/backstage-plugin-quay run start
 ```
+
+## For users
+
+### Using Quay plugin in Backstage
+
+Quay is a frontend plugin that enables you to view the information about the container images.
+
+#### Prerequisites
+
+- Your Backstage application is installed and running.
+- You have installed the Quay plugin.
+
+  For installation process, see [Installation](#installation).
+
+#### Procedure
+
+1. Open your Backstage application and select a component from the **Catalog** page.
+1. Go to the **Image Registry** tab.
+
+   The **Image Registry** tab in the Backstage UI contains a list of container images and related information, such as **TAG**, **LAST MODIFIED**, **SECURITY SCAN**, **SIZE**, **EXPIRES**, and **MANIFEST**.
+
+   ![quay-tab](./images/quay-plugin-backstage1.png)
+
+1. If a container image does not pass the security scan, select the security scan value of the image to check the vulnerabilities.
+
+   ![quay-tab](./images/quay-plugin-backstage2.png)
+
+   The vulnerabilities page displays the associated advisory with a link, severity, package name, and current and fixed versions.
+
+   ![quay-tab-vulnerabilities](./images/quay-plugin-backstage3.png)
+
+   The advisory link redirects to the Red Hat Security Advisory page containing detailed information about the advisory, including the solution.
```

Comment:
Just a nitpick, this would make it easier for me to read:
```suggestion
   The advisory link redirects to the Red Hat Security Advisory page that contains detailed information about the advisory, including the solution.
```

Reply:
Done